python/ctsm/modify_fsurdat/fsurdat_modifier.py

```diff
@@ -0,0 +1,115 @@
+"""
+Run this code by using the following wrapper script:
+tools/modify_fsurdat/fsurdat_modifier
+
+The wrapper script includes a full description and instructions.
+"""
+
+import sys
+from configparser import ConfigParser
+from ctsm.utils import get_config_value, select_value
+from ctsm.modify_fsurdat.modify_fsurdat import ModifyFsurdat
+
+
+def main ():
+    """
+    Description
+    -----------
+    Calls function that modifies an fsurdat (surface dataset)
+    """
+
+    # read the config file
+    config = ConfigParser()
+    cfg_path = './modify.cfg'
+    config.read(cfg_path)
+
+    # required: user must set these in ./modify.cfg
+    fsurdat_in = get_config_value(config, 'modify_input', 'fsurdat_in',
+                                  cfg_path)
+    fsurdat_out = get_config_value(config, 'modify_input', 'fsurdat_out',
+                                   cfg_path)
+
+    # not required: user may set these in ./modify.cfg
+    # TODO Error checking not complete for these. Should it be done here
+    # while reading or later when using them?
+    temp = get_config_value(config, 'modify_input', 'idealized', cfg_path,
+                            allowed_values=['True','False','UNSET'])
```

Comment:
@billsacks this may need follow-up. I addressed your...
- first checkbox by using the renamed constant in utils.py `CONFIG_UNSET` and
- second checkbox by replacing `get_config_value` with `getboolean`.